### PR TITLE
Try adding zlib

### PR DESCRIPTION
**Pull request by RaulPPelaez**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by RaulPPelaez** [comment]
@conda-forge-admin, please rerender

**Comment by h-vetinari** [comment]
> perhaps libzlib is redundant?

Yup, that's the case, because `zlib` depends on it already (and will add a run-export to that effect).

File: recipe/meta.yaml [hunk 1]
```diff
@@ -38,6 +38,7 @@ requirements:
     - pip
     - llvm
     - libmlir
+    - zlib
     - libzlib
```
Comment (h-vetinari) [inline recipe/meta.yaml:42]:
```suggestion
```

**Review by h-vetinari — APPROVED** [review]
Thanks for the PR!

**Pull request merged**